File: fizzbuzz.py
def retourne_fizzbuzz(entree):
	sortie = str(entree)
	if entree % 15 == 0:
		sortie = "fizzbuzz"
	elif entree % 3 == 0:
		sortie = "fizz"
	elif entree % 5 == 0:
		sortie = "buzz"
	return sortie

File: FizzBuzz.hs
import Test.Hspec

data FBType = Same Integer | Fizz deriving (Eq, Show)

fizzBuzz :: Integer -> FBType
fizzBuzz 3 = Fizz
fizzBuzz x = Same x

main = hspec $ do
  describe "FizzBuzz should" $ do
    it "return 1" $
      (fizzBuzz 1) `shouldBe` Same 1
    it "return Fizz" $
      (fizzBuzz 3) `shouldBe` Fizz
